packages/polygon/src/Polygon.test.ts

```diff
@@ -0,0 +1,139 @@
+import { GreaterThanOrEqual, apply } from '@rabbitholegg/questdk/filter'
+import { describe, expect, test } from 'vitest'
+import { bridge } from './Polygon.js'
+import { POLYGON_CHAIN_ID, ETH_CHAIN_ID, CHAIN_ID_ARRAY } from './chain-ids.js'
+import { MAINNET_BRIDGE } from './contract-addresses.js'
```

Comment:
Any reason for importing all these files with `.js` extensions included

Reply:
Compiler was complaining when I didn't have them on there, here's the error:
```
Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean './types.js'
```

Lmk if there's a smarter way to be fixing this, I agree it looks weird _especially_ since they're `.ts` files

Reply:
ESM modules require you to specify the file extension